core/processing.go

```diff
@@ -137,7 +137,16 @@ func (c *Core) dispatching(bp BundlePack) {
 		return
 	}
 
-	if c.HasEndpoint(bp.MustBundle().PrimaryBlock.Destination) {
+	bndl, err := bp.Bundle()
+	if err != nil {
+		log.WithFields(log.Fields{
+			"bundleID": bp.Id,
+			"error":    err.Error(),
+		}).Debug("Error dispatching bundle")
```

Comment:
If something errors and aborts the processing pipeline, the log level should be at least *info*.

Reply:
I guess the real question is whether this should ever happen during normal operation - if yes, then info does only generate log-spam, if not, then it's probably a warning if not fatal.